vim25/soap/client.go

```diff
@@ -196,9 +199,8 @@ func (c *Client) NewServiceClient(path string, namespace string) *Client {
 		u.Host = vc.Host
 	}
 
-	client := NewClient(u, c.k)
+	client := newClientWithTransport(u, c.k, t)
 	client.Namespace = "urn:" + namespace
-	client.DefaultTransport().TLSClientConfig = c.DefaultTransport().TLSClientConfig
 	if cert := c.Certificate(); cert != nil {
```

Comment:
Is it safe to also remove this line as well now since the certificate ultimately originates from the transport?

```
certs := c.t.TLSClientConfig.Certificates
```